website/sidebars.js

```diff
@@ -135,6 +135,7 @@ export default {
       items: [
         "history-and-hard-forks/intro-to-hard-fork",
         "history-and-hard-forks/ckb-hard-fork-history",
+        "history-and-hard-forks/history-vm-version",
```

Comment:
Should we also mention this article in tech-explanation/ckb-vm?

Reply:
updated